depmapomics/qc/config.py

```diff
@@ -59,11 +65,16 @@
 
 LINES_TO_DROP_COMMON = {"ACH-001108", "ACH-001011", "ACH-001092"}
 LINES_TO_DROP_DNA = LINES_TO_DROP_COMMON
-LINES_TO_DROP_RNA = LINES_TO_DROP_COMMON
+LINES_TO_DROP_RNA = LINES_TO_DROP_COMMON | {"ACH-002778", "ACH-002745"}
 LINES_TO_DROP = {"DNA": LINES_TO_DROP_DNA, "RNA": LINES_TO_DROP_RNA}
 
 
-LINES_TO_RELEASE_SHEET = "https://docs.google.com/spreadsheets/d/1YuKEgZ1pFKRYzydvncQt9Y_BKToPlHP-oDB-0CAv3gE/edit#gid=1929030925"
+# original
+# LINES_TO_RELEASE_SHEET = "https://docs.google.com/spreadsheets/d/1YuKEgZ1pFKRYzydvncQt9Y_BKToPlHP-oDB-0CAv3gE/edit?usp=sharing"
+
+# copy of original with 21Q4v2 and 22Q1 combined into 22Q1
+LINES_TO_RELEASE_SHEET = "https://docs.google.com/spreadsheets/d/1IO9GqeU8m3S_PJb4U2VCKbePVrxBA-k-7jp4A3bhxd8/edit?usp=sharing"
```

Comment:
we don't do this anymore, right? 
we should want to delete all the logic related to the lines to release or the ccle sample tracker

Reply:
@5im1z